scripts/travis/build.sh

```diff
@@ -44,11 +44,12 @@ OS=$("${SCRIPTPATH}/../ostype.sh")
 ARCH=$("${SCRIPTPATH}/../archtype.sh")
 
 # Get the go build version.
-GOLANG_VERSION=$(./scripts/get_golang_version.sh)
-
-curl -sL -o ~/gimme https://raw.githubusercontent.com/travis-ci/gimme/master/gimme
-chmod +x ~/gimme
-eval "$(~/gimme "${GOLANG_VERSION}")"
+if [ -z "${NO_GIMME}" ]; then
```

Comment:
It looks like this is only used for windows, what wasn't working? According to the docs windows should be supported.

Reply:
I can't remember the exact error, but when I was having issues with it on circleci's windows machines. I could go ahead and retract this change since we're planning to get windows working after merging these changes

Reply:
I don't really mind the flag, but maybe name it something like `SKIP_GO_INSTALL`